pastas/utils.py

```diff
@@ -542,11 +545,11 @@ def remove_console_handler(logger=None):
             logger.removeHandler(handler)
 
 
-def add_file_handlers(logger=None, filenames=('info.log', 'errors.log'),
-                      levels=(logging.INFO, logging.ERROR), maxBytes=10485760,
-                      backupCount=20, encoding='utf8',
-                      fmt='%(asctime)s - %(name)s - %(levelname)s - %(message)s',
-                      datefmt='%y-%m-%d %H:%M'):
+def add_file_handlers(logger: Optional[Type[Any]] = None, filenames: Optional[Tuple[str, str]] = ('info.log', 'errors.log'),
+                      levels: Optional[Tuple[Any, Any]] = (logging.INFO, logging.ERROR), maxBytes: Optional[int] = 10485760,
```

Comment:
I think this can just be Optional[Tuple[Any]], it doesn't necessarily contain only two items?

Reply:
agreed, removed